v2/emailpassword/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -4,20 +4,72 @@ title: 2. Backend config
 hide_title: true
 ---
 
+<!-- COPY SECTION -->
+<!-- ./emailpassword/serverless/with-aws-lambda/backend-config.mdx -->
+<!-- 1 -->
+
 import AppInfoForm from "/src/components/appInfoForm"
 import CoreInjector from "/src/components/coreInjector"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
 
 # 2. Backend config
 
-## 1) Install supertokens package
+## 1) Create Lambda layer for required libraries
+
+<BackendSDKTabs>
```

Comment:
Using this also adds a Golang tab. Modify this component to take a prop which tells it to not show golang and then give that prop here.

Reply:
Done.